minecraft/bukkit/src/main/java/dev/triumphteam/cmd/bukkit/BukkitCommandManager.java

```diff
@@ -146,13 +146,13 @@ public void unregisterCommand(@NotNull final BaseCommand command) {
         // TODO add a remove functionality
     }
 
-    public void setBasePermission(@NotNull final String basePermission) {
-        this.basePermission = basePermission;
+    public void setBasePermissions(@NotNull final String[] basePermissions) {
```

Comment:
This and the getter shouldn't be here, it should be immutable and passed only once to the constructor. You can leave this part as well, something I can implement later.
Basically works like a global prefix to all permissions.

Reply:
They actually are entirely unused in the codebase, so they can be removed :)